README.md

```diff
@@ -111,39 +111,46 @@ The first step is to create our pipeline in Spark. For our example we will manua
 
 
 ```scala
-import org.apache.spark.ml.feature.{StringIndexerModel, Binarizer}
+import ml.combust.bundle.BundleFile
+import ml.combust.mleap.spark.SparkSupport._
+import org.apache.spark.ml.Pipeline
+import org.apache.spark.ml.bundle.SparkBundleContext
+import org.apache.spark.ml.feature.{Binarizer, StringIndexer}
+import org.apache.spark.sql._
+import org.apache.spark.sql.functions._
+import resource._
 
-// User out-of-the-box Spark transformers like you normally would
-val stringIndexer = new StringIndexerModel(uid = "si", labels = Array("hello", "MLeap")).
-  setInputCol("test_string").
-  setOutputCol("test_index")
+  val datasetName = "./mleap-docs/assets/spark-demo.csv"
```

Comment:
This should probably point to the `examples` folder where we add the spark-demo.csv file.

Reply:
Updated.